<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddManualItemFormClass</class>
 <widget class="QDialog" name="AddManualItemFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddManualItemForm</string>
  </property>
  <widget class="QFrame" name="frameCurrentItem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>321</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameColorSample">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>205</width>
     <height>511</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QPushButton" name="pushButtonEliminateColor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminate color</string>
    </property>
    <property name="icon">
     <iconset resource="PropertyDynamicMaskingPI.qrc">
      <normaloff>:/Qtres/ColorDelete.png</normaloff>:/Qtres/ColorDelete.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddColor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Color</string>
    </property>
    <property name="icon">
     <iconset resource="PropertyDynamicMaskingPI.qrc">
      <normaloff>:/Qtres/ColorAdd.png</normaloff>:/Qtres/ColorAdd.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>340</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>この輝度範囲がマスクされる</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxEnsmall">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>収縮ドット数</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>膨張ドット数</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxEnlarge">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>450</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSetMergin">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>このマージンを設定</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyDynamicMaskingPI.qrc">
     <normaloff>:/Qtres/Setting.png</normaloff>:/Qtres/Setting.png</iconset>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxMergin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>膨張ドット数</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>480</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>設定</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyDynamicMaskingPI.qrc">
     <normaloff>:/Qtres/Setting.png</normaloff>:/Qtres/Setting.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>キャンセル</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyDynamicMaskingPI.qrc">
     <normaloff>:/Qtres/Cancel.png</normaloff>:/Qtres/Cancel.png</iconset>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>spinBoxMergin</tabstop>
  <tabstop>spinBoxEnsmall</tabstop>
  <tabstop>spinBoxEnlarge</tabstop>
  <tabstop>pushButtonSetMergin</tabstop>
  <tabstop>pushButtonOK</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonAddColor</tabstop>
  <tabstop>pushButtonEliminateColor</tabstop>
 </tabstops>
 <resources>
  <include location="PropertyDynamicMaskingPI.qrc"/>
 </resources>
 <connections/>
</ui>
